QtAwesome/iconic_font.py

```diff
@@ -10,12 +10,20 @@
 _default_options = {
     'color': QColor(50, 50, 50),
     'scale_factor': 0.9,
+    'rotation': False,      # Valid options are: 'spin', or 'pulse'
+    'parent': False,        # Ref to instance of widget holding the icon
+    'spin_interval': 10,    # miliseconds
+    'spin_step': 1,         # degrees
+    'pulse_interval': 300,  # miliseconds
+    'pulse_step': 45,       # degrees
```

Comment:
Yes I also considered this, but on stacked icons it would be a dict inside a dict inside a list... so it felt a bit obfuscated.

Still, pulse and spin should have default values of step and speed, so where should I store this then?